fiftyone/server/lightning.py

```
@@ -376,12 +376,13 @@ async def _do_async_query(
     if match_filter:
         query.insert(0, {"$match": match_filter})
 
-    return [i async for i in collection.aggregate(query)]
+    cursor = await collection.aggregate(query)
+    return [i async for i in cursor]
 
 
 async def _do_distinct_queries(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_do_distinct_queries`

(ANN202)